pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -302,6 +303,19 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 	m.lastObservations = observations
 	m.Unlock()
 
+	// the only candidates viable for eviction are those pods that had anything running.
+	activePods := podFunc()
+	if len(activePods) == 0 {
+		glog.Errorf("eviction manager: eviction thresholds have been met, but no pods are active to evict")
+		return
+	}
+
+	// evict pods if there is a resource uage violation from local volume temporary storage
+	// If eviction happenes in localVolumeEviction function, skip the rest of eviction action
+	if m.localVolumeEviction(activePods) {
```

Comment:
@vishh @derekwaynecarr, any preference as to whether doing local volume evictions is part of the same control loop as the rest of evictions?  The main advantage of having them together is to not perform other evictions when pods are already being evicted for exceeding their local volume storage space